src/common/checksums.cpp

```diff
@@ -142,20 +142,21 @@ QByteArray makeChecksumHeader(const QByteArray &checksumType, const QByteArray &
     return header;
 }
 
-QByteArray findBestChecksum(const QByteArray &checksums)
+QByteArray findBestChecksum(const QByteArray &_checksums)
```

Comment:
@mgallien Adding underscore before the parameter diverges from our coding style. Also, other functions don't have parameters starting from `_` in this file. So, I am wondering why we need to change it here.

Reply:
that is a valid point but I guess I would prefer to do the clean up once we have finished handling all the backlog of cherry-pick
@allexzander are you OK with that ?

Reply:
doing it now will make the next cherry-pick harder and harder

Reply:
@allexzander what do you think ?

Reply:
@mgallien Ok. I see. It's the reason I try not to do too much refactoring in my PRs either. Let's keep it like that.